### ISSUE TITLE
Bug fix release 7.1.1

### ISSUE THREAD

**Issue by kitchoi** [enthought/traitsui#1419]
Several bug fixes have gone into master since 7.1.0 was released. It might be good to have a bug fix release 7.1.1.
See PRs which may be backported: https://github.com/enthought/traitsui/pulls?q=is%3Apr+is%3Aclosed+label%3A%22need+backport+to+7.1%22

Pre-release
---
- [x] Check and label PRs that need to be backported to 7.1
- [x] Create a new branch from `maint/7.1`, cherry pick PRs labeled with 'need backport to 7.1' to it. e.g. `get cherry-pick -x [merge-commit-hash]`, update changelog, then open a PR against `maint/7.1`. (#1421)
- [x] Once this PR is merged, remove the labels from the backported PRs
- [x] Test against other ETS packages and other traitsui-using projects
- [x] Check MANIFEST and requirements are still up to date.
- [x] Test building the documentation

Release
-------
- [x] Create a new branch from the `maint/7.1` branch. (#1423)
        - Set release to `7.1.1`, and set `IS_RELEASED` is true; commit
        - Install from source distribution and run tests again. `git clean -ffxd`, `python setup.py sdist`, `pip install dist/traitsui-version.tar.gz[test]`, `cd anywhere-not-the-traitsui-repo-root`, `python -m unittest discover traitsui`
        - Open a PR targeting `maint/7.1`
- [x] Once the previous PR is merged, create another PR targeting `maint/7.1` (#1424)
        - Bump the version to `7.1.2` and set `IS_RELEASED` to false; commit
- [x] Tag (annotated!) the release commit (where `IS_RELEASED` is true), and push the tag (e.g. `git tag -a -m "Release <release version number>" <release version number> <commit-hash>`, `git push origin <release version number>`)
- [x] Make PR targeting `gh-pages` branch: Generate documentation and copy the content to the branch. Verify that the resulting docs/index.html looks good. (#1425)
- [x] Upload to PyPI
- [x] Test the PyPI package
- [x] Announcement

Post-release
-------------
- [x] Package update for `enthought/free` repository (for EDM)
- [x] Backport release note and change log to master (#1426)

**Comment by kitchoi** [comment]
The release is now available on PyPI and EDM. Announcement has been made. Closing.
